### DISCUSSION TITLE
error log  when use php mysqli insert data

### DISCUSSION THREAD

**Discussion by chenshangwei** [apache/doris#6619] [A - General / Q&A]
Message: errCode = 2, detailMessage = Failed to commit txn 49598. Tablet [10193] success replica num 0 is less than quorum replica num 1 while error backends 10002

Message: errCode = 2, detailMessage = wait close failed. NodeChannel[10208-10002] add batch req success but status isn't ok, load_id=5ebfdf4f91d9470f-bd78e922b4933057, txn_id=2676, backend id=10002:8060, errmsg=tablet writer write failed, tablet_id=10209, txn_id=2676, err=-235

**Comment by EmmyMiao87** [comment]
err=-235 is a common mistake. 
What he meant was that there are too many versions of this tablet that can no longer be loaded.

1. First, you need to check the number of versions of this tablet through the ```show tablet {tablet id}``` command and ``` show proc xxx```. If it is around 500, it is indeed the problem.
2. Open the url of ```compactionstatus``` to check the version merge situation, for example, is it because the load(insert, stream load etc) frequency is too high? Or delete and load are interspersed.
3. If the version merging speed cannot keep up just because the load frequency is too high, you only need to adjust the usage method. 
!! Doris recommends the import method to be batch load.
